frontend/packages/console-app/console-extensions.json

```
@@ -76,5 +76,52 @@
       },
       "provider": { "$codeRef": "actions.useDeploymentActionsProvider" }
     }
+  },
+  {
+    "type": "console.user-preference/group",
+    "properties": {
+      "id": "general",
+      "label": "%console-app~General%",
+      "insertBefore": "language"
+    }
+  },
+  {
+    "type": "console.user-preference/group",
+    "properties": {
+      "id": "language",
+      "label": "%console-app~Language & region%",
+      "insertBefore": "notifications",
+      "insertAfter": "general"
+    }
+  },
+  {
+    "type": "console.user-preference/group",
+    "properties": {
+      "id": "notifications",
+      "label": "%console-app~Notifications%",
+      "insertAfter": "language"
+    }
+  },
+  {
+    "type": "console.user-preference/item",
+    "properties": {
+      "id": "console.preferredPerspective",
+      "label": "%console-app~Perspective%",
+      "groupId": "general",
+      "description": "%console-app~If a perspective is not selected, the console defaults to the last viewed.%",
+      "field": {
+        "type": "dropdown",
+        "userSettingsKey": "console.preferredPerspective",
+        "defaultValue": "latest",
```

Similar to ALL_NAMESPACES_KEY to ensure that we don't have an issue with a perspective (and use a similar way when doing this for namespaces later). :)

```suggestion
        "defaultValue": "%LATEST%",
```

---

Do we even need a value here? If the value is undefined, can we treat it as latest?